src/query/functions/aggregation/absent_test.go

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package aggregation
+
+import (
+	"math"
+	"testing"
+	"time"
+
+	"github.com/m3db/m3/src/query/block"
+	"github.com/m3db/m3/src/query/executor/transform"
+	"github.com/m3db/m3/src/query/models"
+	"github.com/m3db/m3/src/query/parser"
+	"github.com/m3db/m3/src/query/test"
+	"github.com/m3db/m3/src/query/test/executor"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func toArgs(f float64) []interface{} { return []interface{}{f} }
+
+var (
+	start     = time.Now()
+	testBound = models.Bounds{
+		Start:    start,
+		Duration: time.Hour,
+		StepSize: time.Minute * 15,
+	}
+)
+
+func mustMakeMeta(tags ...string) block.Metadata {
+	return block.Metadata{
+		Bounds: testBound,
+		Tags:   test.MustMakeTags(tags...),
+	}
+}
+
+func mustMakeSeriesMeta(tags ...string) block.SeriesMeta {
```

Comment:
Aren't these available already as test.MustMakeSeriesMeta and test.MustMakeMeta?

Reply:
Unfortunately adds a circular test dependency on `test` and `block`, is there a decent way of resolving it? Tried `block_test` package but don't think that's importable from other packages right?